### PR TITLE
Create popinga.it.xml

### PR DESCRIPTION
**Pull request by pamatt**

### CODE REVIEW

**Comment by pamatt** [comment]
Ruleset for popinga.it domain

File: src/chrome/content/rules/popinga.it.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="Popinga-it"> 
+  <target host="popinga.it" /> 
+  <target host="www.popinga.it" /> 
+  <rule from="^http:" to="https:" />
```
Comment (J0WI) [inline src/chrome/content/rules/popinga.it.xml]: Please use tabs to indent lines.

**Comment by pamatt** [comment]
I changed the patch using tabs for indentation

File: src/chrome/content/rules/popinga.it.xml (hunk 1 above)
Comment (pamatt) [inline src/chrome/content/rules/popinga.it.xml]: Done, thanks

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**